app/src/main/java/org/gdg/frisbee/android/appwidget/UpcomingEventWidgetProvider.java

```diff
@@ -123,7 +123,18 @@ public void onNotFound(String key) {
                 });
 
         }
-
+        private Event getNextEvent(List<Event> eListEvents) {
```

Comment:
I think because there are merge conflicts. @evindj could you use the current develop branch and add your changes on top?